### DISCUSSION TITLE
draft closed by dropping all commits

### DISCUSSION THREAD

**Discussion by Worldwidebrine** [community/community#38057] [Pull Requests]
Reproduce:
1. create pull request draft with commits
2. drop all commits
3. force-push

Observed behavior:
The draft is closed

Is this intended?

**Comment by david-engelmann** [comment]
The draft is closed after pushing an empty pull request?

**Reply by Worldwidebrine** [reply]
Yes

**Comment by airtower-luna** [comment]
Well, at that point the head branch is effectively merged already, so it makes sense. :sweat_smile:

**Reply by Worldwidebrine** [reply]
But it's a draft.
